rotkehlchen/assets/resolver.py

```diff
@@ -82,14 +98,26 @@ def resolve_asset(
     def get_asset_type(identifier: str) -> AssetType:
         # TODO: This is ugly here but is here to avoid a cyclic import in the Assets file
         # Couldn't find a reorg that solves this cyclic import
+        from rotkehlchen.constants.assets import CONSTANT_ASSETS  # pylint: disable=import-outside-toplevel  # isort:skip  # noqa: E501
         from rotkehlchen.globaldb.handler import GlobalDBHandler   # pylint: disable=import-outside-toplevel  # isort:skip  # noqa: E501
 
         instance = AssetResolver()
         cached_data = instance.types_cache.get(identifier)
         if cached_data is not None:
             return cached_data
 
-        asset_type = GlobalDBHandler().get_asset_type(identifier)
+        try:
+            asset_type = GlobalDBHandler().get_asset_type(identifier)
+        except UnknownAsset:
+            if identifier not in CONSTANT_ASSETS:
+                raise
+
+            log.debug(f'Attempt to get asset_type for asset with {identifier} using the packaged database')  # noqa: E501
```

Comment:
```suggestion
            log.debug(f'Attempt to get asset_type for {identifier} using the packaged database')  # noqa: E501
```

A general tip: When you write for `asset with {identifier}` this will render: `for asset with 42432324324` ... which makes no sense as you don't see what that value is. Since by default we use identifiers to describe them in logs you don't need to write it.